From: Himanshu Madhani <hmadhani@marvell.com>
Date: Wed, 12 Feb 2020 13:44:35 -0800
Subject: scsi: qla2xxx: Use QLA_FW_STOPPED macro to propagate flag
Patch-mainline: v5.7-rc1
Git-commit: 0fc5b7e62322e1aa8384013ee1caff3f87d30207
References: bsc#1157424

This patch uses QLA_FW_STOPPED macro so that flag is propogated to all the
QPairs.

Link: https://lore.kernel.org/r/20200212214436.25532-25-hmadhani@marvell.com
Signed-off-by: Himanshu Madhani <hmadhani@marvell.com>
Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
Acked-by: Daniel Wagner <dwagner@suse.de>
---
 drivers/scsi/qla2xxx/qla_init.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/scsi/qla2xxx/qla_init.c
+++ b/drivers/scsi/qla2xxx/qla_init.c
@@ -6676,7 +6676,7 @@ qla2x00_abort_isp_cleanup(scsi_qla_host_
 	ha->flags.n2n_ae = 0;
 	ha->flags.lip_ae = 0;
 	ha->current_topology = 0;
-	ha->flags.fw_started = 0;
+	QLA_FW_STOPPED(ha);
 	ha->flags.fw_init_done = 0;
 	ha->chip_reset++;
 	ha->base_qpair->chip_reset = ha->chip_reset;
